lib/ca_testing/drivers/v4/options.rb

```diff
@@ -1,12 +1,16 @@
 # frozen_string_literal: true
 
 require "selenium/webdriver/common/options"
-
 require "selenium/webdriver/chrome/options"
 require "selenium/webdriver/firefox/options"
-require "selenium/webdriver/edge/options"
 require "selenium/webdriver/safari/options"
 
+begin
+  require "selenium/webdriver/edge/options"
+rescue LoadError
+  require "selenium/webdriver/edge_chrome/options"
```

Comment:
Probably worth writing a comment here saying this is needed due to the paths being inconsistent during the v4 alphas/betas